.github/workflows/pr_checks.yml

```diff
@@ -16,8 +16,16 @@ jobs:
       - name: Install dependencies
         run: pip install -r requirements_dev.txt
 
-      - name: Run Black
-        run: black -l 99 --check .
+      - name: Install make
+        run: |
+          apt-get update
+          apt-get install make
+
+      - name: Run Black Check
+        run: make black-check
+
+      - name: Run Linters
```

Comment:
I think it would be good to improve the name of the whole job. _"Check code formatting"_ does not suggest that `pylint` will be triggered. Maybe something like: _"Check code quality (formatting and linters)"_? 🙂 

The second idea is to run it separately but only if run in parallel, to not increase the time of the pipeline 😉

Reply:
Its fast enough for now - those steps actually making it mare parallel at this point could slow it down - overall process.